index.html

```diff
@@ -64,15 +64,15 @@ <h1>Bikes for Refugees</h1>
               Providing donated bikes and accessories to refugees and asylum
               seekers in Scotland.
             </p>
-            <button>Donate a bike today</button>
-            <button>Volunteer</button>
+            <button class="donateBtn">Donate a bike today</button>
+            <button class="volunteerBtn">Volunteer</button>
           </div>
 
           <div>
             <h2 class="heading-underline">Learn more</h2>
 
-            <div>
-              <div class="article">
+            <div class="articles">
```

Comment:
```suggestion
            <section class="articles">
```
Think maybe a `section` tag is a bit more semantic than a `div` https://stackoverflow.com/questions/6939864/what-is-the-difference-between-section-and-div

Reply:
Thanks @Simon2828 for reviewing my work. I will apply all changes soon.